### PR TITLE
chore: add a docker-compose for a postgresql db

### PR DESCRIPTION
**Pull request by phiz71**

### CODE REVIEW

File: docker/quick-setup/postgresql/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+networks:
+  frontend:
+    name: frontend
+  storage:
+    name: storage
+
+volumes:
+  data-elasticsearch:
+  data-mongo:
```
Comment (ytvnr) [inline docker/quick-setup/postgresql/docker-compose.yml]: can be deleted ?

File: docker/quick-setup/postgresql/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+services:
+  mongodb:
+    image: mongo:${MONGODB_VERSION:-3.6}
+    container_name: gio_apim_mongodb
+    restart: always
+    volumes:
+      - ./.data/db-mongo:/data/db
+      - ./.logs/apim-mongodb:/var/log/mongodb
+    networks:
+      - storage
```
Comment (ytvnr) [inline docker/quick-setup/postgresql/docker-compose.yml]: same ?

File: docker/quick-setup/postgresql/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+      - elasticsearch
+    volumes:
+      - ./.logs/apim-gateway:/opt/graviteeio-gateway/logs
+      - ./config/jdbc-driver:/opt/graviteeio-gateway/plugins/ext/repository-jdbc
+    environment:
+      - gravitee_management_type=jdbc
+      - gravitee_management_jdbc_url=jdbc:postgresql://pg_database:5432/gravitee
+      - gravitee_management_jdbc_username=postgres
+      - gravitee_management_jdbc_password=password
+      - gravitee_ratelimit_mongodb_uri=mongodb://mongodb:27017/gravitee?serverSelectionTimeoutMS=5000&connectTimeoutMS=5000&socketTimeoutMS=5000
```
Comment (ytvnr) [inline docker/quick-setup/postgresql/docker-compose.yml]: maybe we can keep jdbc for ratelimit ?

**Review by ytvnr — COMMENTED** [review]
questions about rate limit

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-ssvrdpbhbh.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/add-postgres-quick-setup/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-xkukoapgbv.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/add-postgres-quick-setup/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: docker/quick-setup/postgresql/docker-compose.yml (hunk 1 above)
Comment (ytvnr) [reply docker/quick-setup/postgresql/docker-compose.yml]: can still be deleted ? :)

**Review by ytvnr — COMMENTED** [review]
Can you delete the jar from this commit ?
we do not want to have that kind of stuff in our repo, imho, we should just have a .`driver` folder, add it in `gitignore` and just document that user have to add the driver properly.

wdyt ?

**Review by ytvnr — APPROVED** [review]

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-gyrbtlagrc.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/add-postgres-quick-setup/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Review by marcambier — APPROVED** [review]

**Pull request merged**